templates/cocktail/home.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+
+    <h1>Type in your cocktail <img src="{{ asset('styles/osrs-cocktail.png') }}"
```

Comment:
Use new line, would look more readable:
`<h1>
    Type in your cocktail
    <img src="{{ asset('styles/osrs-cocktail.png') }}"/>
</h1>
`